src/gdriveApi.js

```diff
@@ -197,3 +198,35 @@ async function shareItem(id, shareData) {
 
   logger.log('verbose', 'gdrive - shareItem - done', id, shareData)
 }
+
+export async function getForeignCurrencyRates() {
+  const monthNames = ['Jan', 'Feb', 'Mar', 'Apr', 'May', 'Jun', 'Jul', 'Aug', 'Sep', 'Oct', 'Nov', 'Dec']
+  logger.log('verbose', 'gdrive - read spreadsheet')
+
+  // source google sheet for up-to-date currency rates
+  // https://docs.google.com/spreadsheets/d/1HfaU9jXZtCfdotEEfDO0d4EpDLyhmqdepaWgn_a6Lsw/edit#gid=1807750274
+  const fileId = '1HfaU9jXZtCfdotEEfDO0d4EpDLyhmqdepaWgn_a6Lsw'
+  const today = new Date()
+  const sheetName =  `${monthNames[today.getMonth() - 1]} ${today.getFullYear()}` // e.g. 'Nov 2022'
+
+  const {data:{values:rates}} = await sheets.spreadsheets.values.get({
+    spreadsheetId: fileId,
+    range: `${sheetName}!A2:B50`,
+  })
+
+  const formatToFixed = (num, decimals) => Number.parseFloat(num).toFixed(decimals)
```

Comment:
`.toFixed` may be wrong for rounding some numbers due to JS number representation. just bringing up it, not sure how important the correct math here is.
example: `parseFloat('1.005').toFixed(2)` yields `'1.00'` (you can try e.g. in browser console)
as we have lodash in this repo, its `round` function may be the easiest upgrade of this code.